data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/parse/AbstractParseProcessor.java

```diff
@@ -1,22 +1,12 @@
-/*
```

Comment:
Please include the copyright headers on existing or new files. See https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#license-headers.

Reply:
Oops, thanks for catching this. IntelliJ was hiding them so I didn't realize.